polyfills/Object/hasOwn/config.toml

```
@@ -0,0 +1,26 @@
+aliases = ["es2022"]
+dependencies = [
+	"_ESAbstract.CreateMethodProperty",
+	"_ESAbstract.HasOwnProperty",
+	"_ESAbstract.ToObject",
+	"_ESAbstract.ToPropertyKey",
+]
+docs = "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn"
+spec = "https://tc39.es/ecma262/#sec-object.hasown"
+
+[browsers]
+android = "*"
+bb = "*"
+chrome = "<93"
+edge = "<93"
+edge_mob = "<93"
+firefox = "<92"
```

```suggestion
chrome = "<93"
edge = "*"
edge_mob = "*"
firefox = "<92"
```

`edge` with `chromium` is normalised to `chrome` so these entries can be simplified.
_This is not blocking_